### ISSUE TITLE
Have Calls to Extension Functions Respect `maxCallsInOneLine`

### ISSUE THREAD

**Issue by adamsmd** [saveourtool/diktat#1705]
## Describe the bug

The number of allowed extension function calls on a line does not seem to respect `[WRONG_NEWLINES].maxCallsInOneLine`.

## Steps to Reproduce

In `diktat.kt`:
```yaml
- name: WRONG_NEWLINES
  configuration:
    maxParametersInOneLine: 100
    maxCallsInOneLine: 100
```

Somewhere in your Kotlin code:
```kotlin
3.toInt().toInt().toInt().toInt()
```

(Granted, using `toInt()` repeatedly this way is dumb.  This is just to make the example simple.)

## Expected behavior

No warning from DiKTat.

## Observed behavior

DiKTat produces the following warning:

```
[WRONG_NEWLINES] incorrect line breaking: wrong split long `dot qualified expression` or `safe access expression` (diktat-ruleset:newlines)
```

## Notes

This issue seems to be related to the use of a magic number instead of a configuration variable at the following, but I don't understand enough of the code to know if that number controls other things:
https://github.com/saveourtool/diktat/blob/9c96ec43e36818a325bb43e4ef12ceffa838828a/diktat-rules/src/main/kotlin/com/saveourtool/diktat/ruleset/rules/chapter3/files/NewlinesRule.kt#L160

## Environment information
* diktat version: 1.2.5
* build tool (maven/gradle): gradle
* how is diktat run (CLI, plugin, etc.): plugin
* kotlin version: 1.8
* operating system: Linux
* link to a project (if your project is public): Project not public yet
